roles/horizon/tasks/monitoring.yml

```diff
@@ -1,6 +1,12 @@
 ---
-- name: apache2 process check
-  sensu_process_check: service=apache2
```

Comment:
can we use a ternary here, something like:   `"service={{ ( os == 'ubuntu' )| ternary('apache2', 'httpd') }}"`